packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -130,10 +136,14 @@ const BpkCalendarNav = props => {
         >
           <ArrowRightIcon className={getClassName('bpk-calendar-nav__icon')} />
           <span className={getClassName('bpk-calendar-nav__text--hidden')}>
-            {formatMonth(addMonths(baseMonth, 1))}
+            {nextMonthLabel}
           </span>
         </button>
       </div>
+
+      <BpkAriaLive>
+        {`${nowDisplayingLabel} ${formatMonth(baseMonth)}`}
```

Comment:
Though aria-hidden *works* with BpkAriaLive on Android, it doesn't on iOS. In order to remove the aria-hidden and have the month read out all the time, the label text was made more descriptive to prevent confusion.